provider/pkg/gen/python-templates/helm/v2/helm.py

```diff
@@ -576,12 +591,16 @@ def _parse_chart(all_config: Tuple[str, Union[ChartOpts, LocalChartOpts], pulumi
     file = open(overrides, 'w')
     pulumi.Output.all(file, data).apply(_write_override_file)
 
+    apiversions_arg = []
+    if config.api_versions is not None:
+        apiversions_arg = [f'--api-versions={version}' for version in config.api_versions]
```

Comment:
```suggestion
    apiversions_arg = [f'--api-versions={version}' for version in config.api_versions] if config.api_versions else []
```

Reply:
Aha -- I was trying to put the conditional inside the comprehension and it wasn't working. Didn't remember that you can put it outside like that.